ci/desktop.groovy

```diff
@@ -172,11 +172,12 @@ def compileMacOS() {
 def bundleMacOS(type = 'nightly') {
   def pkg = common.pkgFilename(type, 'dmg')
   dir(packageFolder) {
-    sh 'git clone https://github.com/vkjr/StatusAppFiles.git'
-    sh 'unzip StatusAppFiles/StatusIm.app.zip'
-    sh 'cp -r assets/share/assets StatusIm.app/Contents/MacOs'
-    sh 'chmod +x StatusIm.app/Contents/MacOs/ubuntu-server'
-    sh 'cp ../desktop/bin/StatusIm StatusIm.app/Contents/MacOs'
+    sh 'curl -L -O "https://github.com/gnl/StatusAppFiles/raw/master/StatusIm.app.zip"'
```

Comment:
I really dislike this step. We have to fix this at some point.
Also, why are we not using the repo anymore?

Reply:
Not a fan of it either. I moved away from cloning the repo because if there are multiple commits, as soon there will be, cloning it will take up a lot more space than necessary. This way we only get the one file.